src/dmd/aggregate.h

```diff
@@ -42,11 +42,11 @@ enum class Baseok : uint8_t
     semanticdone  // all base classes semantic done
 };
 
-enum StructPOD
+enum class ThreeState : uint8_t
 {
-    ISPODno,            // struct is not POD
-    ISPODyes,           // struct is POD
-    ISPODfwd            // POD not yet computed
+    none,         // value not yet computed
```

Comment:
I think *Neither* is more descriptive here, perhaps *NotYet* would be better still (i.e. it's more obvious what it means when read downstream)

Reply:
`unknown` is a good option, based on https://dlang.org/phobos/std_typecons.html#Ternary.